aea/cli/packages.py

```diff
@@ -171,6 +172,21 @@ def lock_packages(ctx: Context, check: bool) -> None:
         raise click.ClickException(str(e)) from e
 
 
+@package_manager.command(name="init")
+def _init_packages_repo() -> None:
+    """Initialize packages repository."""
+
+    packages_dir = Path.cwd() / PACKAGES
+    if packages_dir.exists():
+        raise click.ClickException(
```

Comment:
Is this an intended behaviour? I think it will be very troublesome. From a user's perspective what I want is "Initialize the local registry IF it does not exist, and use the existing one if it exists." I would show an information message if it exists "found local registry at ..."

Moreover the output of command autonomy init should show information about this. eg.
remote registry in use = ....  
local registry path = ...
author = ...
version = ...

Reply:
Let's not make this more complicated, right now we only want to provide support initialising an empty local repository. These local repositories are project specific, whereas the information like author, version and remote registry address are global settings. 

> I would show an information message if it exists "found local registry at ..."

Yes this command does this exactly, the click exceptions don't get raised as normal exceptions they just output the error message and exits with error code being 1

Reply:
I still see this solution may cause usability problems... For example, if a user works in 2 different projects and needs to switch between them, it will throw an exception when he wants to re-init a project after switching....

Please collect other engineer's opinion (e.g., @solarw ) and then decide.

Reply:
OK, I apologise, I realized the command is `aea packages init`, (I was thinking of `aea init`)

Then seems OK.